doc/projektivelho.md

```diff
@@ -1,5 +1,89 @@
 # Projektivelho
 
+## Yleistä
+
+ProjektiVelho-integraatio on tiedostojen käsin uploadaamisen ohella toinen tapa tuoda rata-aineistoja Geoviitteeseen.
```

Comment:
Rivien pituudet saattavat olla vähän jänniä. Tämä sen vuoksi, että kirjoitin suurimman osan tästä dokumentaatiosta GitHubin sisäänrakennetulla markdown-editorilla, joka ei näytä rivien pituuksia. Voin säätää ne kuntoon IDEAssa jos niissä on jotain vikaa ja koette nillittämisen arvoiseksi

Reply:
Hmm, ideassa taitaa meillä olla päällä formatointi-optio päällä niin että se rivittää automaattisesti. Tästä aiheutunee se että kun seuraavan kerran joku koskee tiedostoon, kaikki rivitykset hyppii. Eli siinä mielessä olisi hyvä tehdä formatointi kuntoon. Itse pituuksissa ei ole mitään ongelmaa, kyllä tuo ihan selkeältä näyttää.

Edit: vai olikohan tuo optio vain minulla? Jos se ei ole yhteisissä asetuksissa päällä niin unohda koko juttu :)

Reply:
Testasin ja mulla tuo asetus ei näyttäisi olevan päällä, joten ei taida olla ainakaan meidän projektikonffiksessa. Jätän nämä nyt toistaiseksi näin, koska a) ainakaan mulla IDEA ei vaikuttaisi osaavan rivittää Markdownia automaagisesti muutenkaan, b) rivit eivät kumminkaan mitään superpitkiä ole ja koko massan läpikäynti olisi Työmaa^TM (pisin rivi on nykyään 121 merkkiä, joka on omasta mielestäni vielä ihan säädyllinen) ja c) sekvenssikaaviossakin on jo yli 100 merkin rivejä, enkä tiedä olisiko niitä edes mahdollista rivittää

Reply:
Tai no, rivitinpäs nämä sittenkin alle 100 merkkiin. Ei tuo niin iso työmaa ollutkaan